packages/client/src/workflow-client.ts

```diff
@@ -799,6 +832,22 @@ export class WorkflowClient extends BaseClient {
         (info as unknown as WorkflowExecutionDescription).raw = raw;
         return info;
       },
+      async fetchHistory() {
+        let nextPageToken: Uint8Array | undefined = undefined;
+        const history = Array<temporal.api.history.v1.IHistoryEvent>();
+        for (;;) {
+          const response: temporal.api.workflowservice.v1.GetWorkflowExecutionHistoryResponse =
+            await this.client.workflowService.getWorkflowExecutionHistory({
+              nextPageToken,
+              namespace: this.client.options.namespace,
+              execution: { workflowId, runId },
+            });
+          history.push(...(response.history?.events ?? []));
+          nextPageToken = response.nextPageToken;
+          if (nextPageToken == null || nextPageToken.length == 0) break;
```

Comment:
nit.

We should probably have a lint rule for this.
```suggestion
          if (nextPageToken == null || nextPageToken.length === 0) break;
```

Reply:
Right. Actually copied that code elsewhere without paying too much attention, so I'll definitely add a lint rule.